src/pools/index.ts

```diff
@@ -90,6 +93,8 @@ export function parseNewPool(
                 newPool = GyroEPool.fromPool(pool);
             }
         } else if (pool.poolType === 'FX') newPool = FxPool.fromPool(pool);
+        else if (pool.poolType === 'Managed')
+            newPool = KassandraManagedPool.fromPool(pool);
```

Comment:
This is the only part that I don't have a clear answer at the moment. 
In case anyone else creates a new Managed Pool with different config from yours, the SOR might break.
Are you guys planning on releasing any subsequent Managed Pool?
If yes, is there anything in common with the current one that we can use as a second identifier? Maybe the `owner` address?
If not, we could even use the `poolId` as an intermediate solution until we need to develop a more generalized approach.